spec/unit/provider/package/rpm_spec.rb

```diff
@@ -52,6 +52,14 @@
       expect { @provider.run_action(:any) }.to raise_error(Chef::Exceptions::Package)
     end
 
+    it "should accept uri formed source" do
+      @new_resource.source "http://example.com/ImageMagick-c++-6.5.4.7-7.el6_5.x86_64.rpm"
+      allow(@provider).to receive(:popen4).and_return(@status)
```

Comment:
`popen4` is used not only at here, and I did not touch the codes. Can you update it after merging this if you care of it?